tests/wpt/metadata/html/dom/dynamic-markup-insertion/opening-the-input-stream/016.html.ini

```diff
@@ -1,15 +1,8 @@
 [016.html]
   type: testharness
-  expected: TIMEOUT
   [Timeout on original window, scope]
-    expected: NOTRUN
-
-  [Timeout on original window, this object]
-    expected: NOTRUN
+    expected: FAIL
 
   [Timeout on new window, scope]
-    expected: NOTRUN
-
-  [Timeout on new window, this object]
-    expected: NOTRUN
+    expected: FAIL
```

Comment:
Is this related to https://github.com/whatwg/html/issues/1698 or something else?